doc/upload-block-datavolume.md

```diff
@@ -0,0 +1,72 @@
+# How to upload an image to a block raw PV User Guide
```

Comment:
I think it's almost identical to the original upload doc. Maybe just add there a section about block mode?

Reply:
Sounds good. I will remove this file and open a new PR to update doc/Upload.md